client/state/jetpack-connection-health/selectors/get-jetpack-connection-health.js

```diff
@@ -0,0 +1,16 @@
+import 'calypso/state/jetpack-connection-health/init';
+
+/**
+ * Returns true if the current site Jetpack connection was checked and is unhealthy
```

Comment:
This description doesn't seem to be accurate; the selector actually returns the jetpack connection health information instead.